src/geophires_x/Units.py

```diff
@@ -350,3 +353,19 @@ class Decay_RateUnit(str,Enum):
 class Inflation_RateUnit(str,Enum):
     """Decay rate Units"""
     KPASCALPERYEAR = "kPa/yr"
+
+
+class JobsPerEnergyUnit(str,Enum):
+    """Jobs per energy Units"""
+    JOBSPERKW = "jobs/kW"
+    JOBSPERMW = "jobs/MW"
+    JOBSPERGW = "jobs/GW"
+
+class RoyaltyPerEnergyUnit(str,Enum):
+    """Royalty per energy Units"""
+    ROYALTYPERMW = "MUSD/MW"
+    ROYALTYPERKW = "MUSD/kW"
```

Comment:
This is redundant and/or inconsistent with `EnergyCostUnit`

Reply:
I don't think so, but I will check.